backend/app/models/xata.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.getXataClient = exports.XataClient = void 0;
+// Generated by Xata Codegen 0.23.5. Please do not edit.
+const client_1 = require("@xata.io/client");
+/** @typedef { import('./types').SchemaTables } SchemaTables */
+/** @type { SchemaTables } */
+const tables = [
+  {
+    name: "books",
+    columns: [
+      { name: "title", type: "string" },
+      { name: "description", type: "text" },
+      { name: "available", type: "bool" },
+    ],
+  },
+];
+/** @type { import('../../client/src').ClientConstructor<{}> } */
+const DatabaseClient = (0, client_1.buildClient)();
+const defaultOptions = {
+  databaseURL: "https://demo-uni3q8.us-east-1.xata.sh/db/books",
```

Comment:
The database URL should come from env vars

Reply:
Upon testing, I can see the file is generated using the `xata` CLI tool.

Reply:
Will accept the PR as is and will handle this on the Bunnyshell side, together with the creation of the Xata dedicated component.

Reply:
Yeah, it's a generated file, I was thinking if it makes sense to commit that file or not. It might make sense for the `databaseURL` to be fixed and only the `XATA_BRANCH` to come via env. Let me know if you hit any issues or need help.

Reply:
If you do need to pass the `databaseURL` via env, you can override it here: https://github.com/bunnyshell/demo-books/pull/24/files#diff-12c37c60d9e9842a51db733a4b9234a0f25780fe2e1c4849426f4eb5e01aeba9R9 See here for the available options https://xata.io/docs/typescript-client/configuration